### PR TITLE
Comment Fixes

### PR DESCRIPTION
**Pull request by seamuslee001**

### CODE REVIEW

File: api/v3/ActivityType.php [hunk 1]
```diff
@@ -110,10 +110,9 @@ function _civicrm_api3_activity_type_create_spec(&$params) {
  *
  * @param array $params
  *   Array including id of activity_type to delete.
- *
- * @return array
- *   API result array
- *
+ * @return array API result array
+ * API result array
```
Comment (eileenmcnaughton) [inline api/v3/ActivityType.php]: phpstorm does this annoying thing....

File: api/v3/MembershipStatus.php [hunk 2]
```diff
@@ -120,8 +120,9 @@ function civicrm_api3_membership_status_update($params) {
  * This API is used for deleting a membership status
  *
  * @param array $params
- *
```
Comment (eileenmcnaughton) [inline api/v3/MembershipStatus.php]: Something or other insists on this line after @param for me..

File: api/v3/Contribution.php [hunk 3]
```diff
@@ -544,6 +544,7 @@ function _civicrm_api3_contribution_completetransaction_spec(&$params) {
  * @param array $params
  *   Input parameters.
  *
+ * @return mixed
```
Comment (eileenmcnaughton) [inline api/v3/Contribution.php]: hopefull an array?

File: api/v3/ActivityType.php [hunk 4]
```diff
@@ -110,10 +110,8 @@ function _civicrm_api3_activity_type_create_spec(&$params) {
  *
  * @param array $params
  *   Array including id of activity_type to delete.
- *
- * @return array
- *   API result array
- *
+ * @return array API result array
```
Comment (eileenmcnaughton) [inline api/v3/ActivityType.php]: hmm - I think you actually want that indent formatting for some reason - perhaps phpcs will fail on it

File: api/v3/Contribution.php (hunk 3 above)
Comment (seamuslee001) [inline api/v3/Contribution.php]: I think the ipn function return is the issue here

File: api/v3/ActivityType.php (hunk 4 above)
Comment (seamuslee001) [inline api/v3/ActivityType.php]: I ran this against civilint and it survived it

Comment (eileenmcnaughton) [inline api/v3/ActivityType.php]: OK - I'm happy with merge on pass to cover that

Comment (eileenmcnaughton) [inline api/v3/ActivityType.php]: Ah - it might be because I have stricter drupal stds in my phpstorm -anyway - it'll do

**Pull request merged**